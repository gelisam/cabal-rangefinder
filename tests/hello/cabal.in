name:                hello
version:             0.1.0.0
build-type:          Simple
cabal-version:       >=1.10

library
  build-depends:       base >= 4.6,
                       containers -any,

                       mtl
  exposed-modules:     Hello
