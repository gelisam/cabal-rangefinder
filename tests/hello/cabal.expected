name:                hello
version:             0.1.0.0
build-type:          Simple
cabal-version:       >=1.10

library
  build-depends:       base >=4.6,
                       containers -any,

                       mtl >=2.1.2
  exposed-modules:     Hello
